### PR TITLE
Update en_pt

### PR DESCRIPTION
**Pull request by thelegitdolt**
last time i swear

### CODE REVIEW

File: src/main/resources/assets/upgrade_aquatic/lang/en_pt.json [hunk 1]
```diff
@@ -1,667 +1,667 @@
...
+  "subtitles.upgrade_aquatic.entity.pike.flop": "Pikefish marooned",
+  "subtitles.upgrade_aquatic.entity.pike.hurt": "Pikefish hurts",
+  "subtitles.upgrade_aquatic.entity.pike.bite": "Pikefish bites",
+  "subtitles.upgrade_aquatic.entity.pike.spit": "Pikefish spits",
+  "tooltip.upgrade_aquatic.amur_pike": "Pikefish o' Silk",
+  "tooltip.upgrade_aquatic.redfin_pike": "Redfin'd Pikefish",
+  "tooltip.upgrade_aquatic.brown_northern_pike": "Brown Pikefish",
+  "tooltip.upgrade_aquatic.mahogany_northern_pike": "Dark Pikefish",
+  "tooltip.upgrade_aquatic.jade_northern_pike": "Pikefish o' Jewels",
+	"tooltip.upgrade_aquatic.spotted_jade_northern_pike": "Pikefish o' Jewels w' Dots o' Polka",
```
Comment (SmellyModder) [inline src/main/resources/assets/upgrade_aquatic/lang/en_pt.json:628]: Not indented consistently with the rest of the entries.

File: src/main/resources/assets/upgrade_aquatic/lang/en_pt.json [hunk 2]
```diff
@@ -1,667 +1,667 @@
...
+
+	"effect.upgrade_aquatic.restfulness": "Wide Awake",
+	"item.minecraft.potion.effect.restfulness": "Clap o' Thunder",
+	"item.minecraft.potion.effect.restfulness_strong": "Clap o' Thunder",
+	"item.minecraft.splash_potion.effect.restfulness": "Grenade o' Clappin' Thunder",
+	"item.minecraft.splash_potion.effect.restfulness_strong": "Grenade o' Clappin' Thunder",
+	"item.minecraft.lingering_potion.effect.restfulness": "Bottled Fog o' Clappin' Thunder",
+	"item.minecraft.lingering_potion.effect.restfulness_strong": "Bottled Fog o' Clappin' Thunder",
+	"item.minecraft.tipped_Bolt.effect.restfulness": "Bolt o' Clappin' Thunder",
+	"item.minecraft.tipped_Bolt.effect.restfulness_strong": "Bolt o' Clappin' Thunder",
```
Comment (SmellyModder) [inline src/main/resources/assets/upgrade_aquatic/lang/en_pt.json:561]: These lines are not indented consistently with the rest of the entries.

File: src/main/resources/assets/upgrade_aquatic/lang/en_pt.json [hunk 3]
```diff
@@ -1,667 +1,667 @@
...
+    "item.upgrade_aquatic.mulberry_pie": "Tart o' River Capers",
+
+    "item.upgrade_aquatic.boiled_purple_pickerelweed": "Boil'd Mermaid Weed o' Purple",
+    "item.upgrade_aquatic.boiled_blue_pickerelweed": "Boil'd Mermaid Weed o' Blue",
+    "item.upgrade_aquatic.pike": "Raw Pikefish",
+    "item.upgrade_aquatic.cooked_pike": "Smoked Pikefish",
+    "item.upgrade_aquatic.lionfish": "Raw Red Devil",
+    "item.upgrade_aquatic.cooked_lionfish": "Smoked Red Devil",
+    "item.upgrade_aquatic.perch": "Raw Perch",
+    "item.upgrade_aquatic.cooked_perch": "Smoked Perch",
```
Comment (SmellyModder) [inline src/main/resources/assets/upgrade_aquatic/lang/en_pt.json:505]: These lines are not indented consistently with the rest of the entries.

**Review by SmellyModder — CHANGES_REQUESTED** [review]

**Comment by thelegitdolt** [comment]
made a new pull request that fixes these errors with some more lines updated!

**Pull request closed without merging**